<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PinConfig</class>
 <widget class="QWidget" name="PinConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>668</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="layoutG" rowstretch="7,7,0,0" columnstretch="5,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>20</number>
   </property>
   <item row="1" column="1">
    <widget class="PinTypeHelper" name="widget_PinTypeHelper" native="true"/>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select board</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CenteredCBox" name="comboBox_board"/>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>16</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="CurrentConfig" name="widget_currConfig" native="true"/>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="layoutV_pins">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CenteredCBox</class>
   <extends>QComboBox</extends>
   <header>centered_cbox.h</header>
  </customwidget>
  <customwidget>
   <class>CurrentConfig</class>
   <extends>QWidget</extends>
   <header>currentconfig.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PinTypeHelper</class>
   <extends>QWidget</extends>
   <header>pintypehelper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
